### PR TITLE
[Translation zh-tw] Fundamentals > Events

### PR DESCRIPTION
**Pull request by kennith**
[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/zh-TW/review/433)

### CODE REVIEW

File: src/content/zh-tw/events/_index.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
```
Comment (petele) [inline src/content/zh-tw/events/_index.yaml]: Extra YAML `---` is added

File: src/content/zh-tw/events/_index.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    heading: 找你附近的活動
+    items:
+    - description: |-
+        <p>
+        活動由Google員工，本地Google開發人員小組或其他開發者聚會運行。 每個事件都有一點不同，但通常情況下，一個事件可以包括一個技術講座，一系列的代碼實驗室，以及個人編碼的時間。 在你的地區找到一個或開始你自己的！
+        </p>
+        <p>
+        如果您無法找到附近的活動，請不要擔心！ 一切
+           材料是可用的，所以你可以組織你自己的事件或
+           通過家裡的內容工作。 <a href="\"#featured-events\"">按入精選活動部分了解更多詳情</a>.
```
Comment (petele) [inline src/content/zh-tw/events/_index.yaml]: Overly escaped quotes.

**Comment by petele** [comment]
@agektmr, @kennith, @limhenry - it looks like GitLocalize is adding extra quotes and YAML elements to this file

**Comment by agektmr** [comment]
@sotayamashita @chikathreesix can you take a look?

**Comment by sotayamashita** [comment]
@agektmr I'll check it out.

**Comment by petele** [comment]
Any updates on this?

File: src/content/zh-tw/events/_index.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+book_path: "/web/events/_book.yaml"
+landing_page:
+  header:
+    description: 參加的開發人員活動並開始黑客
```
Comment (xuhang57) [inline src/content/zh-tw/events/_index.yaml]: “黑客” means hacker not hacking. Maybe just use the word `hacking` here so it makes more sense.

**Comment by sotayamashita** [comment]
@petele Sorry not to replay fast. Unfortunately, we cannot do something on this PR because base repository is unknown which means deleted. 

@kennith Is it possible to re-create PR?

**Comment by kennith** [comment]
@sotayamashita I manually create a new PR #5887 .

**Pull request closed without merging**